vms/platformvm/block/executor/helpers_test.go

```diff
@@ -81,9 +81,19 @@ var (
 	genesisBlkID ids.ID
 	testSubnet1  *txs.Tx
 
+	// Node IDs of genesis validators. Initialized in init function
+	genesisNodeIDs []ids.NodeID
+
 	errMissing = errors.New("missing")
 )
 
+func init() {
+	for range preFundedKeys {
+		nodeID := ids.GenerateTestNodeID()
+		genesisNodeIDs = append(genesisNodeIDs, nodeID)
+	}
+}
+
```

Comment:
nit: we should pre-allocate this

```suggestion
func init() {
	genesisNodeIDs = make([]ids.NodeID, len(preFundedKeys))
	for i, _ := range preFundedKeys {
		genesisNodeIDs[i] = ids.GenerateTestNodeID()
	}
}
```

Reply:
yup, done